### PR TITLE
Purrfect toast

### PR DESCRIPTION
**Pull request by CJCCyril**

### CODE REVIEW

**Comment by MatouTechnologies** [comment]
This pull request has been deployed to Vercel.
<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://matou-technologies-purrfect-ui-storybook-65.vercel.app'>https://matou-technologies-purrfect-ui-storybook-65.vercel.app</a></td>
  </tr>
  <tr>
    <td><strong>🔍 Inspect:</strong></td>
    <td><a href='https://vercel.com/matou-technologies/purrfect-ui/59XWcWmni4ke4mWepHVJUNjWNsvc'>https://vercel.com/matou-technologies/purrfect-ui/59XWcWmni4ke4mWepHVJUNjWNsvc</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="deploy-preview-storybook"></hidden>

File: src/feedback/Toast/Toast.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+const ButtonsWrapper = styled(Container)`
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+`;
+
+const Buttons = () => {
+  const toaster = useToaster();
+
+  return (
```
Comment (Rcaradec) [inline src/feedback/Toast/Toast.stories.tsx:51]: If we want to be consistent, you should add a Title here ;-) 
`import { Title } from "src/utils/StorybookComponents/Titles";
`
`<Title>Toast</Title>`

File: src/feedback/Alert/Alert.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      options: alertKindValues,
+      mapping: { alertKindValues },
+      control: { type: "select" },
+      description: "",
+    },
+  },
+};
+
+export const AlertList = () => {
+  return (
```
Comment (Rcaradec) [inline src/feedback/Alert/Alert.stories.tsx:52]: If we want to be consistent, you should add a Title here ;-)
`import { Title } from "src/utils/StorybookComponents/Titles"; 
`
`<Title>Toast</Title>`

File: src/feedback/Alert/Alert.tsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+const alertKindStyle = (kind: AlertKind, severity: AlertSeverity) => css`
+  background-color: ${"outlined" === kind && theme.color.white};
+  color: ${"outlined" === kind && theme.color[severity]};
+`;
+
+const AlertWrapper = styled.div<{ severity: AlertSeverity; kind: AlertKind }>`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 16px;
```
Comment (Rcaradec) [inline src/feedback/Alert/Alert.tsx:42]: use `${sizing(16)}` here for rem conversion

File: src/feedback/Alert/Alert.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+  margin: 16px 0;
```
Comment (Rcaradec) [inline src/feedback/Alert/Alert.tsx:43]: use ` ${sizing(16, 0)}` here for rem conversion

File: src/feedback/Alert/AlertCloseable.tsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+const StyledDivider = styled(Divider)`
+  height: unset;
+  align-self: stretch;
+`;
+
+const AlertCloseable = ({ onClose }: { onClose: () => void }) => {
+  return (
+    <>
+      <StyledDivider orientation={"vertical"}></StyledDivider>
```
Comment (Rcaradec) [inline src/feedback/Alert/AlertCloseable.tsx:19]: We don't need a divider here

**Pull request closed without merging**